manifests/init.pp

```diff
@@ -71,6 +71,24 @@
         notify  => Service[$servicename],
         content => template('dhcp/redhat/sysconfig-dhcpd'),
```

Comment:
This file shouldn't be managed if managing the systemd unit file below, put it in an else block.